tests/components/ozw/test_services.py

```diff
@@ -43,34 +47,48 @@ async def test_services(hass, light_data, sent_messages, light_msg, caplog):
     assert msg["payload"] == {"Value": 55, "ValueIDKey": 844425594667027}
 
     # Test set_config_parameter invalid list int
-    await hass.services.async_call(
-        "ozw",
-        "set_config_parameter",
-        {"node_id": 39, "parameter": 1, "value": 12},
-        blocking=True,
-    )
+    with pytest.raises(NotFoundError):
+        assert await hass.services.async_call(
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6285789)